packages/storybook/src/semantic-ui/PlayButton.stories.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React from 'react';
+import PlayButton from '../../../semantic-ui/src/components/PlayButton';
+
+export default {
+  title: 'Components/Semantic UI/PlayButton',
+  component: PlayButton
+};
+
+export const Default = () => (
+  <PlayButton
+    onClick={() => console.log('click!')}
```

Comment:
We can use the `action` method from Storybook Actions instead of `console.log` to automatically log the events in the "Actions" pane.